config/drupal/phpcsfixer.rules.yml

```
@@ -24,15 +24,16 @@ parameters:
                 - try
                 - while
                 - yield
-        braces:
-            allow_single_line_closure: true
-            position_after_functions_and_oop_constructs: same
-            position_after_control_structures: same
-            position_after_anonymous_constructs: same
+        braces_position:
```

does not exists, should be `curly_braces_position`

```suggestion
        curly_braces_position:
```